bootcamp-java/Topic1/classes/ShoppingCart.java

```diff
@@ -0,0 +1,93 @@
+package classes;
+
+import java.util.Date;
+
+public class ShoppingCart {

```

Comment:
Si, lo agrego.

Date: Tue, 20 Jan 2015 06:49:50 -0800
From: notifications@github.com
To: java-bootcamp@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [java-bootcamp] ShoppingCart (#4)

In bootcamp-java/Topic1/classes/ShoppingCart.java:

> @@ -0,0 +1,93 @@
> +package classes;
> +
> +import java.util.Date;
> +
> +public class ShoppingCart {

El ShoppingCart deberia tener una lista de ítems


—
Reply to this email directly or view it on GitHub.

  
    
    
  
  
 		 	   		  =